ui/src/components/WebRTCVideo.tsx

```diff
@@ -106,32 +106,57 @@ export default function WebRTCVideo() {
   );
 
   // Pointer lock and keyboard lock related
-  const isPointerLockPossible = window.location.protocol === "https:";
-
+  const isPointerLockPossible = window.location.protocol === "https:" || window.location.hostname === "localhost";
```

Comment:
Also allowing pointer lock on "localhost" so we can test during development (this will not be an issue in deployed code)

Reply:
Good catch! I didn't realize that it could work on localhost. :-)